### PR TITLE
Create `Bit.Platform.Website` project (#1890)

### LINKED ISSUE #1890: Implement the `BitPlatform` website's initial structure
The `BitPlatform` website's initial structure must be implemented.

### PR DESCRIPTION
**Pull request by msangtarash**
this closes #1890

### CODE REVIEW

File: src/WebSite/Bit.Platform.WebSite/Api/appsettings.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "Logging": {
+        "LogLevel": {
+            "Default": "Information",
+            "Microsoft": "Warning",
+            "Microsoft.Hosting.Lifetime": "Information"
+        }
+    },
+    "AllowedHosts": "*",
+    "UploadPath": "./BitFileUploaderFiles/"
```
Comment (ysmoradi) [inline src/WebSite/Bit.Platform.WebSite/Api/appsettings.json:10]: Please remove this `UploadPath`

File: src/WebSite/Bit.Platform.WebSite/Web/Components/Footer.razor.min.css [hunk 2]
```diff
@@ -0,0 +1 @@
+﻿@font-face{font-family:"Axiforma";src:url("../fonts/Axiforma/woff2/AxiformaExtraBoldItalic.woff2") format("woff2");font-weight:800;}@font-face{font-family:"Axiforma";src:url("../fonts/Axiforma/woff2/AxiformaBold.woff2") format("woff2");font-weight:700;}@font-face{font-family:"Axiforma";src:url("../fonts/Axiforma/woff2/AxiformaSemiBold.woff2") format("woff2");font-weight:600;}@font-face{font-family:"Axiforma";src:url("../fonts/Axiforma/woff2/AxiformaMedium.woff2") format("woff2");font-weight:500;}@font-face{font-family:"Axiforma";src:url("../fonts/Axiforma/woff2/AxiformaRegular.woff2") format("woff2");font-weight:400;}@font-face{font-family:"Axiforma";src:url("../fonts/Axiforma/woff2/AxiformaLight.woff2") format("woff2");font-weight:300;}@font-face{font-family:"Axiforma";src:url("../fonts/Axiforma/woff2/AxiformaThin.woff2") format("woff2");font-weight:100;}.footer{position:absolute;bottom:0;right:0;left:0;width:100%;}
```
Comment (ysmoradi) [inline src/WebSite/Bit.Platform.WebSite/Web/Components/Footer.razor.min.css:1]: Don't push this file. Check git ignore settings

File: src/WebSite/Bit.Platform.WebSite/Web/Components/Footer.razor.css [hunk 3]
```diff
@@ -0,0 +1,42 @@
+﻿@font-face {
```
Comment (ysmoradi) [inline src/WebSite/Bit.Platform.WebSite/Web/Components/Footer.razor.css:1]: Don't push this file. Check git ignore settings

File: src/WebSite/Bit.Platform.WebSite/Web/Components/Footer.razor.cs [hunk 4]
```diff
@@ -0,0 +1,10 @@
+﻿namespace Bit.Platform.WebSite.Web.Components
+{
+    public partial class Footer
+    {
+        protected override void OnInitialized()
```
Comment (ysmoradi) [inline src/WebSite/Bit.Platform.WebSite/Web/Components/Footer.razor.cs:5]: Remove extra override

File: src/WebSite/Bit.Platform.WebSite/Web/Components/Header.razor.cs [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+using Bit.Platform.WebSite.Web.Services;
+using Microsoft.AspNetCore.Components;
+using Microsoft.AspNetCore.Components.Routing;
+
+namespace Bit.Platform.WebSite.Web.Components
+{
+    public partial class Header
+    {
+        protected override void OnInitialized()
+        {
```
Comment (ysmoradi) [inline src/WebSite/Bit.Platform.WebSite/Web/Components/Header.razor.cs:11]: Remove extra override

File: src/WebSite/Bit.Platform.WebSite/Web/Components/Header.razor.css [hunk 6]
```diff
@@ -0,0 +1,163 @@
+﻿@font-face {
```
Comment (ysmoradi) [inline src/WebSite/Bit.Platform.WebSite/Web/Components/Header.razor.css:1]: Don't push this. Check git ignore settings

File: src/WebSite/Bit.Platform.WebSite/Web/Components/Header.razor.min.css [hunk 7]
```diff
@@ -0,0 +1 @@
...in;cursor:pointer;margin-right:8.75rem;}@media screen and (min-width:56.3125em) and (max-width:90em){.header-logo{width:2.25rem;height:1.5rem;margin-right:3rem;}}@media screen and (min-width:34.4375em) and (max-width:56.25em){.header-logo{width:1.875rem;height:1.25rem;margin-right:2rem;}}.github-btn{width:3.0625rem;height:3.0625rem;border-radius:50%;background-image:url("/images/github-icon.svg");background-position:center bottom;background-repeat:no-repeat;background-size:1.875rem;background-color:#001533;cursor:pointer;margin-left:1rem;}.github-btn:hover{background-color:#002a66;}@media screen and (min-width:56.3125em) and (max-width:90em){.github-btn{width:3rem;height:2.375rem;margin-left:1.625rem;background-size:1.6875rem;}}@media screen and (min-width:34.4375em) and (max-width:56.25em){.github-btn{width:2rem;height:1.5rem;margin-left:1.125rem;background-size:.9375rem;}}@media screen and (max-width:34.375em){.github-btn{width:1.25rem;height:1.25rem;background-size:.875rem;}}@media all and (max-width:39.375em){.header-main-container{height:3rem;padding:0 1rem;}.header-logo{width:1.875rem;height:1.25rem;margin-right:.5rem;}.github-btn{width:1.25rem;height:1.25rem;margin-left:0;}}
```
Comment (ysmoradi) [inline src/WebSite/Bit.Platform.WebSite/Web/Components/Header.razor.min.css:1]: Don't push this. Check git ignore settings

File: src/WebSite/Bit.Platform.WebSite/Web/Components/NotFoundComponent.razor.css [hunk 8]
```diff
@@ -0,0 +1,214 @@
+﻿@font-face {
```
Comment (ysmoradi) [inline src/WebSite/Bit.Platform.WebSite/Web/Components/NotFoundComponent.razor.css:1]: Don't push this. Check git ignore settings

File: src/WebSite/Bit.Platform.WebSite/Web/Components/NotFoundComponent.razor.min.css [hunk 9]
```diff
@@ -0,0 +1 @@
...eight:15rem;}@media screen and (min-width:34.4375em) and (max-width:56.25em){.title{width:13.5rem;height:10.5rem;}}@media screen and (max-width:34.375em){.title{width:9rem;height:7rem;}}.description{font-weight:400;font-size:2rem;line-height:3rem;color:#8a8886;text-align:center;}@media screen and (max-width:90em){.description{font-size:1.5rem;line-height:2rem;}}@media screen and (max-width:34.375em){.description{font-size:1.125rem;line-height:1.5rem;}}.disconnected-cable{background-image:url("/images/disconnected-cable.svg");background-size:cover;background-position:center;background-repeat:no-repeat;width:100%;height:5.25rem;}@media screen and (min-width:34.4375em) and (max-width:56.25em){.disconnected-cable{height:4rem;}}@media screen and (max-width:34.375em){.disconnected-cable{height:1.75rem;}}.not-found-back-btn{width:13.3125rem;height:3.5rem;font-weight:600;font-size:1.125rem;line-height:2rem;}@media screen and (min-width:34.4375em) and (max-width:56.25em){.not-found-back-btn{width:10.0625rem;}}@media screen and (max-width:34.375em){.not-found-back-btn{width:7.375rem;}}@media screen and (max-width:56.25em){.not-found-back-btn{height:2.5rem;font-size:.75rem;line-height:1rem;}}
```
Comment (ysmoradi) [inline src/WebSite/Bit.Platform.WebSite/Web/Components/NotFoundComponent.razor.min.css:1]: Don't push this. Check git ignore settings

**Review by ysmoradi — CHANGES_REQUESTED** [review]

**Comment by msynk** [comment]
@ysmoradi 
plz re-review this PR after new changes

File: src/WebSite/Bit.Platform.WebSite/Web/wwwroot/scripts/app.js [hunk 10]
```diff
@@ -0,0 +1,40 @@
+"use strict";
```
Comment (msynk) [inline src/WebSite/Bit.Platform.WebSite/Web/wwwroot/scripts/app.js:1]: this file shouldn't be in the repo.

**Review by msynk — CHANGES_REQUESTED** [review]

**Review by ysmoradi — CHANGES_REQUESTED** [review]
Bit.Platform.WebSite.Web.sln's location is wrong

**Review by msynk — APPROVED** [review]

**Review by ysmoradi — APPROVED** [review]

**Comment by msangtarash** [comment]
PING !

**Pull request merged**